.github/workflows/run-tests.yml

```diff
@@ -19,27 +18,96 @@ jobs:
 
     - name: Create .env file
       run: |
-        echo "REDIS_URL=redis://redis" >> .env
+        echo "REDIS_URL=redis://redis:6379" >> .env
         echo "CONSERVER_API_TOKEN=fake-token" >> .env
+        echo "POSTGRES_HOST=postgres" >> .env
+        echo "POSTGRES_PORT=5432" >> .env
+        echo "POSTGRES_USER=postgres" >> .env
+        echo "POSTGRES_PASSWORD=postgres" >> .env
+        echo "POSTGRES_DB=vcon_test_db" >> .env
+        echo "DATABASE_URL=postgresql://postgres:postgres@postgres:5432/vcon_test_db" >> .env
+        echo "CONSERVER_CONFIG_FILE=/app/test_config.yml" >> .env
 
-    - name: We need to create Conserver network for now
+    - name: Create test configuration file
+      run: |
```

Comment:
@benjigifford I see you also added test_config.yml file in to the repository, in this case you don't need to override it in the Github Action.